data.sql

```diff
@@ -5,4 +5,11 @@ VALUES
 (null,'Agumon', '2020-02-03', 0, true, '10.23'),
 (null,'Gabumon', '2018-11-15', 2, true, '8'),
 (null,'Pikachu', '2021-01-07', 1, false, '15.04'),
-(null,'Devimon', '2017-05-12', 5, true, '11');
\ No newline at end of file
+(null,'Devimon', '2017-05-12', 5, true, '11'),
+(null,'Charmander', '2020-02-08', 0, false, '11'),
+(null,'Plantmon', '2021-11-15', 2, true, '5.7'),
+(null,'Squirtle', '1993-04-02', 3, false, '12.13'),
+(null,'Angemon', '2005-06-12', 1, true, '45'),
+(null,'Boarmon', '2005-06-07', 7, true, '20.4'),
+(null,'Blossom', '1998-10-13', 3, true, '17'),
+(null,'Ditto', '2022-05-14', 4, true, '22');
```

Comment:
- [ ] Some animals have a weight as negative as mentioned in the project requirements. Kindly update that part.
![183878706-9aad8fb2-968c-4641-bae5-7610776caa95](https://user-images.githubusercontent.com/85095702/183890003-50cc830c-dae5-4d06-9df3-7bc6f9f79c92.png)